src/dto/identity.js

```diff
@@ -18,7 +18,7 @@
 // @flow
 
 class IdentityDTO {
-  id: ?string
+  id: string
```

Comment:
!! `id` should be optional, because we set `id` from `Object`, which might not have `id`.
If we want to have `id` as non-optional, we should do some validation first, and fail creating `IdentityDTO` in case `id` is not present.

Reply:
@donce can we just fix this and not wait till he comes back?